src/platform/qt/SettingsView.cpp

```diff
@@ -735,6 +735,13 @@ void SettingsView::saveSetting(const char* key, const QVariant& field) {
 	m_controller->setOption(key, field);
 	m_controller->updateOption(key);
 }
+void SettingsView::loadSetting(const char* key, QAbstractButton* field) {
+	QString option = loadSetting(key);
+	if(option.isNull()){
```

Comment:
Still need that space between `if` and `(`, and now it also needs the space between `)` and `{`.

If you want to contribute to this project, or really any other project, you should take a look at surrounding code and match its style. There's a contributing.md that contains some very basic style rules that were not followed here.